src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Commands/GetAzureContainerRegistryCredential.cs

```diff
@@ -12,7 +12,6 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
```

Comment:
@djyou I think we should keep this `using` and add the `ResourceGroupCompleter` attribute to the `ResourceGroupName` parameter below.

@maddieclayton any idea why the `using` was added but the completer wasn't?

Reply:
I don't think we have implementation for resource group completer. As this is a cleanup PR I would prefer not making functional changes such as adding resource group completer. I meant to remove all unused `using` here and in other places.